src/test/java/jalau/at18/katas/bankocr/sarai/DigitReaderTest.java

```diff
@@ -0,0 +1,33 @@
+package jalau.at18.katas.bankocr.sarai;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class DigitReaderTest {
+    @Test
+    public void acrossNumbersEnumZerro() {
+        String[][] numzero = {
+            {" ","_"," "},
+            {"|"," ","|"},
+            {"|","_","|"} };
+            DigitReader storyzero = new DigitReader(numzero); // envio numzero
+        assertEquals(0, storyzero.getnaturalNumbers());
+    }
+    @Test
+    public void acrossNumbersEnumThree() {
+        String[][] numthree = {
+            {" ","_"," "},
+            {" ","_","|"},
+            {" ","_","|"} };
+            DigitReader storythree = new DigitReader(numthree);
+        assertEquals(3,storythree.getnaturalNumbers());
+    }
+    @Test
+    public void entryNumbers(){
+        String [][] firstEntry = {{" ", "_", " "," ", "|", " "," ", "_", " "," ", "_", " "," ", " ", " "," ", "_", " "," ", "_", " "," ", "_", " "," ", "_", " "," ", "_", " "}};
```

Comment:
You could separate this in multiple lines so it easier to read

Reply:
I changed from matrix to array and that helped me to change the logic.

Reply:
looks good